modules/EvseV2G/v2g_server.cpp

```diff
@@ -293,7 +293,7 @@ static enum v2g_event v2g_handle_apphandshake(struct v2g_connection* conn) {
                    (ev_app_priority >= app_proto->Priority)) {
 
             conn->handshake_resp.supportedAppProtocolRes.ResponseCode =
-                appHand_responseCodeType_OK_SuccessfulNegotiation;
+                appHand_responseCodeType_OK_SuccessfulNegotiationWithMinorDeviation;
```

Comment:
I'm curious to see how real cars deal with this response code. Lets see :sweat_smile: